src/applications/proxy-rewrite/tests/teamsite-header-footer.vrt.spec.js

```diff
@@ -26,7 +28,10 @@ describe.each(TEAMSITES)('%s', (sitename, url) => {
     const element = await page.$(selector);
     await page.waitFor(1000);
     const image = await element.screenshot();
-    expect(image).toMatchImageSnapshot();
+    expect(image).toMatchImageSnapshot({
```

Comment:
`jest-image-snapshot` has per pixel and per comparison threshold settings. By default the per comparison has 0 fault tolerance i.e. it only succeeds if the snapshots are pixel perfect. 

This was something I meant to add months ago but forgot to do a PR for it. Tests intermittently fail on the footer baseline images- there's something about the way chrome renders the VA logo that prevents pixel perfect comparison from passing.